lib/backbone-provider.js

```diff
@@ -0,0 +1,30 @@
+const { Component, PropTypes, Children } = require('react');
+
+class BackboneProvider extends Component {
```

Comment:
To preface, my editor defaults to 2 spaces on new projects, so this wasn't done intentionally. Although I do prefer 2 spaces.

Question though, the rest of the mongodb-js repos use 2 spaces, internally we use 4 spaces.

Happy to do whatever but just wanted to raise that.